src/pages/components/menu.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Menu
+description: >
+  Menus display a list of available choices.
+package: '@zendeskgarden/react-dropdowns'
+propSheets:
+  - dropdowns/src/elements/Menu/Menu.tsx
+  - dropdowns/src/elements/Menu/Separator.tsx
+  - dropdowns/src/elements/Menu/Items/AddItem.tsx
+  - dropdowns/src/elements/Menu/Items/HeaderIcon.tsx
+  - dropdowns/src/elements/Menu/Items/HeaderItem.tsx
+  - dropdowns/src/elements/Menu/Items/Item.tsx
+  - dropdowns/src/elements/Menu/Items/ItemMeta.tsx
+  - dropdowns/src/elements/Menu/Items/MediaBody.tsx
+  - dropdowns/src/elements/Menu/Items/MediaFigure.tsx
+  - dropdowns/src/elements/Menu/Items/MediaItem.tsx
+  - dropdowns/src/elements/Menu/Items/NextItem.tsx
+  - dropdowns/src/elements/Menu/Items/PreviousItem.tsx
+  - dropdowns/src/elements/Menu/Items/PreviousItem.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To display a list of choices, such as options or actions</li>
+    </ul>
+  </Use>
+</Usage>
+
+## Examples
+
+### Select Menu
+
+Works in combination with a [Select](/components/select) to display a list of options.
+
+import MenuSelect from '../../examples/dropdowns/MenuSelect.tsx';
+import MenuSelectCode from '!!raw-loader!../../examples/dropdowns/MenuSelect.tsx';
+
+<CodeExample code={MenuSelectCode}>
+  <MenuSelect />
+</CodeExample>
+
+### Button Menu
+
+Works in combination with a Trigger to display a list of menu items.
+
+import MenuButton from '../../examples/dropdowns/MenuButton.tsx';
+import MenuButtonCode from '!!raw-loader!../../examples/dropdowns/MenuButton.tsx';
+
+<CodeExample code={MenuButtonCode}>
+  <MenuButton />
+</CodeExample>
+
+### Placement
+
+Menu placement can be oriented around a Trigger element in different positions. Default placement
+is to align with the bottom left.
```

Comment:
I updated the example to have `placement="bottom-start"` to match the example. Are there any additional suggestions for this example?

Also, the project tries to avoid "left" vs "right" due to RTL support. Maybe the copy can be reworded to reflect that?

@allisonacs @rossmoody

Reply:
My vote would be to omit the line `Default placement is to align with the bottom left.` or replace with something like "Default placement is `bottom-start`. "

Reply:
I mention it elsewhere, but the example above is the exact same as this one. Feels like perhaps demonstrating a non-default placement might better illustrate what we're talking about?